packages/@react-types/slider/src/index.d.ts

```diff
@@ -7,11 +8,57 @@ export interface BaseSliderProps extends RangeInputBase<number>, LabelableProps,
 }
 
 export interface SliderProps extends BaseSliderProps, ValueBase<number[]> {
-  onChangeEnd?: (value: number[]) => void
+  onChangeEnd?: (value: number[]) => void,
+  direction?: Direction
 }
 
 export interface SliderThumbProps extends AriaLabelingProps, FocusableDOMProps, FocusableProps, Validation, AriaValidationProps, LabelableProps {
   isReadOnly?: boolean,
   isDisabled?: boolean,
-  index: number
+  index: number,
+  direction?: Direction
 }
+
+export interface SpectrumBarSliderBase<T> extends Omit<BaseSliderProps, 'isReadOnly'>, ValueBase<T>, StyleProps {
```

Comment:
I think we can remove `isReadOnly` from the base type

Reply:
You mean completely remove the readonly functionality from aria/stately?

Reply:
Yeah I thought you opened a ticket for that and chungwu confirmed it was ok?

Reply:
Ok 👍